runPatchwork.sh

```diff
@@ -1 +1 @@
-spark-submit --class PatchWorkDemo --master local[4] bin/patchwork_1.0.jar
+spark2-submit --class PatchWorkDemo --master local[4] bin/patchwork_1.1.jar
```

Comment:
To ease reproducility we should provide the `Compound.csv` dataset, required in the demo.

I suggest we:

1. [ ] use an explicit and absolute HDFS path in [PatchWorkDemo.scala, line 23](https://github.com/crim-ca/patchwork/blob/MAJ-spark-2/src/main/scala/example/PatchWorkDemo.scala#L23) : `sc.textFile("hdfs:/datasets/Compound.csv")`

2. [ ] add commands to download and upload the sample dataset to HDFS in ` runPatchwork.sh`. The following worked for me:
  ```bash
  curl http://cs.joensuu.fi/sipu/datasets/Compound.txt | tr "\\t" "," > Compound.csv
  hdfs dfs -mkdir -p /datasets && hdfs dfs -put Compound.csv /datasets/
  ```